packages/gitbook/src/components/DocumentView/CodeBlock/tree.ts

```diff
@@ -0,0 +1,35 @@
+import type { DocumentBlockCode, DocumentInlineAnnotation, DocumentText } from '@gitbook/api';
+
+export type LightNode = (
```

Comment:
I was wondering if we have a better name for it, and if at some point we should merge this as something we could do in `getDocument` to also solve https://linear.app/gitbook-x/issue/RND-6497/optimize-document-and-revision-data-cache-by-reducing-data-size

Reply:
Fine for now